Mapper/BaseGroupedMapper.php

```diff
@@ -70,11 +70,17 @@ public function with($name, array $options = array())
             'collapsed' => false,
             'class' => false,
             'description' => false,
+            'label' => $this->admin->getLabelTranslatorStrategy()->getLabel($name, $this->getName(), 'group'),
             'translation_domain' => null,
             'name' => $name,
             'box_class' => 'box box-primary',
         );
 
+        // BC layer for manual label
+        if (strpos($name, '.')) {
+            $defaultOptions['label'] = $name;
+        }
```

Comment:
Ok I just added the config parameter and use it here. However, has the `BaseGroupedMapper` class is not a service, I rely on this call : https://github.com/sonata-project/SonataAdminBundle/pull/4028/files#diff-d561f27bd57cd565613d211d8d8bb0afR80

I also can inject this parameter when creating the `FormMapper`/`ShowMapper` : 
- via a constructor argument : will make the class not BC
- via a setter : may not be set

What do you think ?